Meadowlark.net/Meadowlark.Net.Core/Backend/Postgresql/Db.cs

```diff
@@ -0,0 +1,101 @@
+using Meadowlark.Net.Core.Backend.Model;
+using Meadowlark.Net.Core.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Npgsql;
+using NpgsqlTypes;
+
+namespace Meadowlark.Net.Core.Backend.Postgresql;
+
+public static class Db
+{
+  // TODO: Get .env loading working
+  private static readonly string connectionString = "Server=localhost;Port=5432;User Id=postgres;Password=abcdefgh1!;Database=MeadowlarkNet;";
```

Comment:
We'll end up using a very different pattern for injecting connectionStrings, likely using Admin API 2.0 as our inspiration.